databuilder/publisher/elasticsearch_publisher.py

```diff
@@ -29,6 +29,9 @@ class ElasticsearchPublisher(Publisher):
     ELASTICSEARCH_ALIAS_CONFIG_KEY = 'alias'
     ELASTICSEARCH_MAPPING_CONFIG_KEY = 'mapping'
 
+    # config to control how many max documents to publish at a time
+    ELASTICSEARCH_DEFAULT_PUBLISHER_STEP = 'publisher_step'
```

Comment:
I wonder if we can use `batch size` instead of `step` for it seems more common term.

By the way, `DEFAULT` usually comes with a value and probably we can name this constant without it.